tests/test_enum.rs

```diff
@@ -86,16 +86,17 @@ fn test_enum_discriminant(input: &[u8], expected: TestEnumDiscriminant) {
     assert_eq!(input.to_vec(), ret_write);
 }
 
+#[derive(PartialEq, Debug, DekuRead, DekuWrite)]
+#[deku(type = "[u8; 3]")]
+enum TestEnumArray {
+    #[deku(id = b"123")]
+    VarA,
+    #[deku(id = "[1,1,1]")]
+    VarB,
+}
+
 #[test]
 fn test_enum_array_type() {
-    #[derive(PartialEq, Debug, DekuRead, DekuWrite)]
```

Comment:
I would prefer to see another struct inside another test function instead of re-using this

Reply:
Refactored all tests for this MR into `tests/test_deku_id.rs`